### PR TITLE
Fix: concept details table layout max-with when long strings

### PR DESCRIPTION
**Pull request by Bilelkihal**
Related issue:
https://github.com/ontoportal-lirmm/bioportal_web_ui/issues/648

Screenshot:
Capture d’écran 2024-06-04 à 10 39 42

### CODE REVIEW

**Review by Bilelkihal — COMMENTED** [review]
✅

File: app/assets/stylesheets/components/concept_details.scss [hunk 1]
```diff
@@ -6,4 +6,12 @@
 
 .concept_details_component table th {
   width: 220px;
+}
+
+.concept_details_component td, .concept_details_component th{
+  max-width: 200px;
+}
+#concepts_json_button{
```
Comment (syphax-bouazzouni) [inline app/assets/stylesheets/components/concept_details.scss:14]: don't use an ID too specific. 
and this file is only for the styling of the concept details component. So does this apply

**Review by syphax-bouazzouni — CHANGES_REQUESTED** [review]

File: app/assets/stylesheets/components/concept_details.scss (hunk 1 above)
Comment (Bilelkihal) [reply app/assets/stylesheets/components/concept_details.scss]: This css is related only to the concepts json button inside the concept details component.

Comment (syphax-bouazzouni) [reply app/assets/stylesheets/components/concept_details.scss]: so put it in `concepts.scss` file, no?

File: app/assets/stylesheets/components/concept_details.scss [hunk 2]
```diff
@@ -6,4 +6,12 @@
 
 .concept_details_component table th {
   width: 220px;
+}
+
+.concept_details_component td, .concept_details_component th{
+  max-width: 200px;
+}
+.concepts_json_button{
```
Comment (syphax-bouazzouni) [inline app/assets/stylesheets/components/concept_details.scss:14]: put this in `concepts.scss`

**Review by syphax-bouazzouni — CHANGES_REQUESTED** [review]

File: app/assets/stylesheets/concepts.scss [hunk 3]
```diff
@@ -165,4 +165,9 @@ div.synonym-change-request button {
 #concepts_json_link{
```
Comment (syphax-bouazzouni) [inline app/assets/stylesheets/concepts.scss:165]: this one is used nowhere

Comment (syphax-bouazzouni) [reply app/assets/stylesheets/concepts.scss:165]: and is the same as your `.concepts_json_button`

**Review by syphax-bouazzouni — CHANGES_REQUESTED** [review]

**Review by syphax-bouazzouni — APPROVED** [review]

File: app/assets/stylesheets/concepts.scss (hunk 3 above)
Comment (syphax-bouazzouni) [reply app/assets/stylesheets/concepts.scss]: done in https://github.com/ontoportal-lirmm/bioportal_web_ui/pull/649/commits/c77c20b7b1434a9cb85e28056f21e41d0830cb06

**Pull request merged**